### PR TITLE
Reviewed all the  changes 

### PR DESCRIPTION
**Pull request by ayushigeorge**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Added and fixed the grammar as suggested
Fixes # (issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: journeys/ayushigeorge31.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Hi!
+I am Ayushi George and I am pursuing Artificial Intelligence and Data Science from JECRC College in Jaipur, Rajasthan.
+I am *GOOGLE DEVELOPMENT STUDENT CLUB LEAD*, STUDENT PLACEMENT COORDINATOR, **STATE VICE PRESIDENT at WICCI**, Web Developer, and Content Writer.
+I have 3 stars on Codechef and 4 stars in MySQ on HackerRank.
+I am an ardent person with immense zeal & zest to work in distinct fields and domains. I am a technical plus creative person who is also a part of robotics club.
+ahead, I am an AWS educate member and spark AR campus ambassador.
+
+
```
Comment (KendallDoesCoding) [inline journeys/ayushigeorge31.md:8]:
```suggestion
```

delete extra lines

File: journeys/ayushigeorge31.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Hi!
+I am Ayushi George and I am pursuing Artificial Intelligence and Data Science from JECRC College in Jaipur, Rajasthan.
+I am *GOOGLE DEVELOPMENT STUDENT CLUB LEAD*, STUDENT PLACEMENT COORDINATOR, **STATE VICE PRESIDENT at WICCI**, Web Developer, and Content Writer.
+I have 3 stars on Codechef and 4 stars in MySQ on HackerRank.
+I am an ardent person with immense zeal & zest to work in distinct fields and domains. I am a technical plus creative person who is also a part of robotics club.
+ahead, I am an AWS educate member and spark AR campus ambassador.
+
```
Comment (KendallDoesCoding) [inline journeys/ayushigeorge31.md:7]:
```suggestion
```

**Review by KendallDoesCoding — COMMENTED** [review]
delete extra lines

**Review by Adarsh-jaiss — APPROVED** [review]

**Review by KendallDoesCoding — APPROVED** [review]

**Comment by KendallDoesCoding** [comment]
Thanks for your contribution!

**Pull request merged**